tests/testthat/setup-load_data.R

```diff
@@ -4,30 +4,31 @@ suppressPackageStartupMessages(library(dbplyr))
+    makers <<- as_tibble(read.table(file = paste0(base_url, "makers.txt"), header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
+    offices <<- as_tibble(read.table(file = paste0(base_url, "offices.txt"), header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
+    orders <<- as_tibble(read.table(file = paste0(base_url, "orders.txt"), header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
+    salary_grades <<- as_tibble(read.table(file = paste0(base_url, "salary_grades.txt"), header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
+    toys <<- as_tibble(read.table(file = paste0(base_url, "toys.txt"), header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
 
-  card_rank     <<- as_tibble(read.table(file = paste0(base_url, "card_rank.txt"),     header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  card_suit     <<- as_tibble(read.table(file = paste0(base_url, "card_suit.txt"),     header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  crayons       <<- as_tibble(read.table(file = paste0(base_url, "crayons.txt"),       header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  customers     <<- as_tibble(read.table(file = paste0(base_url, "customers.txt"),     header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  employees     <<- as_tibble(read.table(file = paste0(base_url, "employees.txt"),     header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  games         <<- as_tibble(read.table(file = paste0(base_url, "games.txt"),         header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  inventory     <<- as_tibble(read.table(file = paste0(base_url, "inventory.txt"),     header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  makers        <<- as_tibble(read.table(file = paste0(base_url, "makers.txt"),        header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  offices       <<- as_tibble(read.table(file = paste0(base_url, "offices.txt"),       header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  orders        <<- as_tibble(read.table(file = paste0(base_url, "orders.txt"),        header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  salary_grades <<- as_tibble(read.table(file = paste0(base_url, "salary_grades.txt"), header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
-  toys          <<- as_tibble(read.table(file = paste0(base_url, "toys.txt"),          header = TRUE, sep = "\t", quote = "", stringsAsFactors = FALSE))
+    db_con <<- DBI::dbConnect(RSQLite::SQLite(), ":memory:")
+    copy_to(db_con, iris, name = "iris_db")
+    iris_db <<- tbl(db_con, "iris_db")
 
-  db_con        <<- DBI::dbConnect(RSQLite::SQLite(), ":memory:")
-  copy_to(db_con, iris, name = "iris_db")
-  iris_db       <<- tbl(db_con, "iris_db")
-
-  iris_dt       <<- lazy_dt(iris)
-  flights_dt    <<- lazy_dt(flights)
```

Comment:
Could you please [configure lintr](https://github.com/jimhester/lintr#project-configuration) to leave these lines as is? Squashing the extra whitespace makes this part much less readable